packages/experiments/src/components/FloatingPicker/Suggestions/Suggestions.tsx

```diff
@@ -18,7 +18,7 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, {}> {
   public currentSuggestion: ISuggestionModel<T> | undefined;
   protected _selectedElement: HTMLDivElement;
   private SuggestionsItemOfProperType: new (props: ISuggestionItemProps<T>) => SuggestionsItem<T> =
-  SuggestionsItem as new (props: ISuggestionItemProps<T>) => SuggestionsItem<T>;
+    SuggestionsItem as new (props: ISuggestionItemProps<T>) => SuggestionsItem<T>;
```

Comment:
Intentional format change?

Reply:
Looks like it was some auto-indentation formatting done by Visual Studio Code. Probably a good change for readability, though